src/aktualizr_info/aktualizr_info_test.cc

```diff
@@ -596,6 +596,59 @@ TEST_F(AktualizrInfoTest, PrintDelegations) {
   }
 }
 
+/**
+ * Verifies aktualizr-info output of snapshot metadata from image repository
+ *
+ * Checks actions:
+ *
+ *  - [x] Print snapshot metadata from image repository
+ */
+TEST_F(AktualizrInfoTest, PrintImageSnapshotMetadata) {
+  Json::Value director_root_json;
+  director_root_json["key-002"] = "value-003";
+  std::string director_root = Utils::jsonToStr(director_root_json);
+  db_storage_->storeRoot(director_root, Uptane::RepositoryType::Director(), Uptane::Version(1));
+
+  Json::Value meta_snapshot;
+  meta_snapshot["signed"]["_type"] = "Snapshot";
+  meta_snapshot["signed"]["expires"] = "2038-01-19T03:14:06Z";
+  meta_snapshot["signed"]["version"] = "2";
+  std::string images_snapshot = Utils::jsonToStr(meta_snapshot);
+  db_storage_->storeNonRoot(images_snapshot, Uptane::RepositoryType::Image(), Uptane::Role::Snapshot());
+
+  aktualizr_info_process_.run(std::vector<std::string>{"--snapshot"});
```

Comment:
Is it really necessary to do an explicit casting here (std::vector<std::string>) ? If not, then it can be removed.